shaders/src/main/glsl/samples/310es/prefix_sum_intfunctions.frag

```diff
@@ -0,0 +1,73 @@
+#version 310 es
+
+/*
+ * Copyright 2019 The GraphicsFuzz Project Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+precision highp float;
+
+layout(location = 0) out vec4 _GLF_color;
+
+uniform vec2 resolution;
+
+void main(void) {
+  uint uselessOutVariable;
```

Comment:
Again, I think built-ins here just take literals; it would be much better if some of them were more complicated by *somehow* using data that comes from (at least one of) loop counters, uniforms, fragCoord, etc.

Reply:
I added injectionSwitch and put it to good use.